src/components/shared/MarkerTooltipContents.js

```diff
@@ -522,10 +534,14 @@ function getMarkerDetails(
         ) {
           return (
             <div className="tooltipDetails">
-              {_markerDetailNullable('url', 'URL', data.URI)}
-              {_markerDetail('pri', 'pri', data.pri)}
-              {_markerDetailNullable('count', 'count', data.count)}
               {_markerDetail('status', 'Status', data.status)}
```

Comment:
Optional: Should this value be more human readable? Right now it displays as `Status: STATUS_START`, which seems pretty opinionated towards our implementation rather than the intent of what it's showing. `Status: Start of request`. I'm not really sure on the exact wording or if we should bother as this PR feels more like an intermediate presentation until the network panel gets nicer.

Reply:
100% in favor of this. Opened #1132